array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -22,21 +22,19 @@ cd $SLURM_SUBMIT_DIR
 ## format paths in config file with project name
 echo "Loading config file for project: " $1
 export PROJECT=$1
-source ./DNAm/config/config.txt 
+source /lustre/projects/Research_Project-MRC190311/DNAm/${PROJECT}/config.txt 
```

Comment:
This assumes the data is always in this folder. Can we revisit the best way to do this?